src/pages/project/[...slug].astro

```diff
@@ -43,7 +43,7 @@ const { title, body, description, roles, pubDate, upDate, featuredImage, previou
   <div class="mt-24 flex w-full items-center justify-between text-lg font-light text-white md:text-xl lg:text-2xl">
     {
       previousProjectSlug ? (
```

Comment:
We can change this to `previousProjectSlug && ...` in order to avoid using ternary operators and having an empty `<p></p>` tag.